test/helpers/cons.go

```diff
@@ -75,16 +75,16 @@ const (
 
 	// Names of commonly used containers in tests.
 
-	Httpd1 = "httpd1"
-	Httpd2 = "httpd2"
-	Httpd3 = "httpd3"
-	App1   = "app1"
-	App2   = "app2"
-	App3   = "app3"
-	Client = "client"
-	Server = "server"
-	Host   = "host"
-
+	Httpd1          = "httpd1"
+	Httpd2          = "httpd2"
+	Httpd3          = "httpd3"
+	App1            = "app1"
+	App2            = "app2"
+	App3            = "app3"
+	Client          = "client"
+	Server          = "server"
+	Host            = "host"
+	HTTPResponse503 = 503
```

Comment:
If you want you can use [`net/http.StatusServiceUnavailable`](https://golang.org/pkg/net/http/#pkg-constants)